### DISCUSSION TITLE
Better integration in GNOME infrastructure

### DISCUSSION THREAD

**Discussion by thibaultamartin** [johnfactotum/foliate#748] [General]
Hello and thanks a lot for all your work on foliate :)

Have you considered moving to [GNOME's Gitlab](https://gitlab.gnome.org)?
It would deepen the integration in the GNOME ecosystem, notably for the translators. Getting your project in GNOME's Gitlab and under the _World_ umbrella would allow it to be integrated automatically in GNOME's translators tool (called [Damned Lies](https://l10n.gnome.org/)).

It's worth noting Gitlab has a tool to import existing GitHub projects.

Thanks, and keep up the good work!

**Comment by itprojects** [comment]
+1

Gnome integration could potentially mean inclusion/assistance from the one group of devs that made and support the GJS/Shell infrastructure, especially since you have flatpak + deb packages through Gnome Builder.

**Comment by itprojects** [comment]
@johnfactotum Also, you can just setup a mirror repository, with some minimal push automation. Some time ago gitlab had a migration spike due to MS purchasing github, so they made ready to use templates.

This way both communities - github and gitlab, can directly be involved.

**Comment by Vanadiae** [comment]
> This way both communities - github and gitlab, can directly be involved.

You can actually login in GNOME gitlab with a github account.

If @johnfactotum is interested, he can apply at https://gitlab.gnome.org/Infrastructure/Infrastructure/-/issues. I think this could be a fairly great idea to move place. :smile_cat:

**Comment by zippy-zebu** [comment]
In my view, simply not worth the hassle. It won't go into gnome rather it will go into world which is rather hidden. And the ui is crowded and complex compared to github. With odd issue tracker. It's gtk project so it can be used under elementary or mate. Translator things brings lot more bugs than solving the issue and I am talking from personal experience. I moved my project from world to here on github and gained lot more contributors & stars and now it's cool.

**Comment by mufeedali** [comment]
> In my view, simply not worth the hassle. It won't go into gnome rather it will go into world which is rather hidden. And the ui is crowded and complex compared to github. With odd issue tracker. It's gtk project so it can be used under elementary or mate. Translator things brings lot more bugs than solving the issue and I am talking from personal experience. I moved my project from world to here on github and gained lot more contributors & stars and now it's cool.

Which project did you move from GNOME's Gitlab to Github? Just asking because I haven't noticed any project like that.
